docs/pages/push-notifications/fcm-credentials.mdx

```diff
@@ -79,11 +80,41 @@ Here are the steps to configure a new Google Service Account Key in EAS for send
 </Step>
 
 <Step label="5">
+  Configure the `google-services.json` file in your app. (Note: you may skip this step if `google-services.json` has already been set up.)
+
+  First, download the `google-services.json` file from Firebase Cloud Console and place it somewhere in your app directory (the root is fine). 
+
+  <ImageSpotlight
+    alt="Download google-services.json from Firebase Cloud Console"
+    src="/static/images/creating-google-service-account/fcm-v1/new-service-account/05-setup-google-services-json.png"
+    style={{ maxWidth: 800 }}
+  />
+
+  Then update your app.json with the key `expo.android.googleServicesFile` and value `path/to/google-services.json`.
```

Comment:
```suggestion
  Then,  in **app.json**, add [`expo.android.googleServicesFile`](/versions/latest/config/app/#googleservicesfile) with its value as the path of the **google-services.json**.
```

- Let's add an internal link to the `android.googleServicesFile` and also mention in words about its value instead of using inline code to highlight that.

A question about this, should we also mention that in case if someone is using Git or any other Version Control System and have public reprository, then, they should add this file to their project's ** .gitignore**? Since this file contains secret values.

Reply:
Good call on the .gitignore, I think that's worth mentioning.